test/onnx/test_onnx_opset.py

```diff
@@ -369,6 +370,31 @@ def forward(self, x):
         x = torch.randn(20, 16, 50)
         check_onnx_opsets_operator(MyDynamicModel(), x, ops, opset_versions=[9, 10])
 
+    def test_grid_sample(self):
+        n, c, h_in, w_in, h_out, w_out = 1, 1, 3, 2, 2, 4
+        ops = [{"op_name": "GridSample"}]
+        ops = {16: ops}
```

Comment:
nit: Avoid reassigning to the same var w/ a different type. Prefer a new name.

Reply:
as not complicated, merged into one line: fc97d87d3fce276cf355f986470cbc2f38ca103f